modules/container-base/README.md

```diff
@@ -0,0 +1,56 @@
+# Dataverse Base Container Image
+
+A "base image" offers you a pre-installed and pre-tuned application server to deploy Dataverse software to.
```

Comment:
Thanks for the review @atrisovic!

I like your idea, but I'm afraid it would not be what is happening here. The resulting image is not containing a single bit of Dataverse software or scripts yet, which I would read from your suggestion. But I see what you are getting at!

Reply:
Thanks for the prompt reply @poikilotherm ! :)
How about:

The Dataverse Base Container Image contains pre-installed and pre-tuned application server that contains the necessary software dependencies for deploying and launching a Dataverse repository installation, but it does not contain the Dataverse software itself.

Reply:
Like that?

```suggestion
The Dataverse Base Container Image contains primarily a pre-installed and pre-tuned application server. That contains the necessary software dependencies for deploying and launching a Dataverse repository installation, but it does not contain the Dataverse software itself.

```

Reply:
Yeah I think so, if it is correct 😄 

I would keep it as a single sentence (but it's just style): 
The Dataverse Base Container Image contains primarily a pre-installed and pre-tuned application server with the necessary software dependencies for deploying and launching a Dataverse repository installation.

Maybe then at the end of the paragraph: The image does not contain the Dataverse software itself.

Reply:
:heavy_check_mark: